src/aiidalab_qe/common/widgets.py

```diff
@@ -617,3 +618,409 @@ def _select_periodicity(self, _=None):
         new_structure.set_pbc(periodicity_options[self.periodicity.value])
         self.structure = None
         self.structure = deepcopy(new_structure)
+
+
+class HubbardWidget(ipw.VBox):
+    """Widget for setting up Hubbard parameters."""
+
+    def __init__(self, input_structure=None):
+        self.input_structure = input_structure
+        self.eigenvalues_help = ipw.HTML(
+            value="For transition metals and lanthanoids, the starting eigenvalues can be defined (Magnetic calculation).",
+            layout=ipw.Layout(width="auto"),
+        )
+        self.hubbard = ipw.Checkbox(
```

Comment:
suggest to change the name, e.g.
```suggestion
        self.activate_hubbard = ipw.Checkbox(
```